diff --git a/node_modules/react-native-ui-datepicker/lib/typescript/datetime-picker.d.ts b/node_modules/react-native-ui-datepicker/lib/typescript/datetime-picker.d.ts
index e454a17..4a02f6b 100644
--- a/node_modules/react-native-ui-datepicker/lib/typescript/datetime-picker.d.ts
+++ b/node_modules/react-native-ui-datepicker/lib/typescript/datetime-picker.d.ts
@@ -22,6 +22,7 @@ export interface DatePickerMultipleProps extends DatePickerBaseProps {
     dates?: DateType[];
     onChange?: MultiChange;
 }
-declare const DateTimePicker: (props: DatePickerSingleProps | DatePickerRangeProps | DatePickerMultipleProps) => JSX.Element;
+export type DateTimePickerProps = DatePickerSingleProps | DatePickerRangeProps | DatePickerMultipleProps
+declare const DateTimePicker: (props: DateTimePickerProps) => JSX.Element;
 export default DateTimePicker;
 //# sourceMappingURL=datetime-picker.d.ts.map
diff --git a/node_modules/react-native-ui-datepicker/lib/typescript/index.d.ts b/node_modules/react-native-ui-datepicker/lib/typescript/index.d.ts
index 9225ee2..49cf569 100644
--- a/node_modules/react-native-ui-datepicker/lib/typescript/index.d.ts
+++ b/node_modules/react-native-ui-datepicker/lib/typescript/index.d.ts
@@ -3,5 +3,6 @@ import './polyfill';
 import DateTimePicker from './datetime-picker';
 export type { DateType, CalendarMode, CalendarDay, CalendarWeek, CalendarMonth, CalendarYear, CalendarComponents, CalendarMonthSelectorProps, CalendarYearSelectorProps, DatePickerBaseProps, } from './types';
 export { useDefaultClassNames, useDefaultStyles } from './theme';
+export type { DateTimePickerProps } from './datetime-picker';
 export default DateTimePicker;
 //# sourceMappingURL=index.d.ts.map
diff --git a/node_modules/react-native-ui-datepicker/src/datetime-picker.tsx b/node_modules/react-native-ui-datepicker/src/datetime-picker.tsx
index 6dc6d66..921b7df 100644
--- a/node_modules/react-native-ui-datepicker/src/datetime-picker.tsx
+++ b/node_modules/react-native-ui-datepicker/src/datetime-picker.tsx
@@ -75,8 +75,10 @@ export interface DatePickerMultipleProps extends DatePickerBaseProps {
   onChange?: MultiChange;
 }
 
+export type DateTimePickerProps = DatePickerSingleProps | DatePickerRangeProps | DatePickerMultipleProps
+
 const DateTimePicker = (
-  props: DatePickerSingleProps | DatePickerRangeProps | DatePickerMultipleProps
+  props: DateTimePickerProps
 ) => {
   const {
     mode = 'single',
diff --git a/node_modules/react-native-ui-datepicker/src/index.ts b/node_modules/react-native-ui-datepicker/src/index.ts
index da2a104..917eb28 100644
--- a/node_modules/react-native-ui-datepicker/src/index.ts
+++ b/node_modules/react-native-ui-datepicker/src/index.ts
@@ -16,4 +16,6 @@ export type {
 } from './types';
 export { useDefaultClassNames, useDefaultStyles } from './theme';
 
+export type { DateTimePickerProps } from './datetime-picker';
+
 export default DateTimePicker;
